src/playground/Weekdays/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { ToggleGroup, ToggleGroupItem } from "@/ui/toggleGroup"
+import { forwardRef } from "react"
+
+interface WeekdaysProps {
+  activatedDays?: string[]
+  daysOfTheWeek?: string[]
+}
+
+const DAYS_OF_THE_WEEK = [
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+  "Sunday",
+]
+
+export const Weekdays = forwardRef<HTMLDivElement, WeekdaysProps>(
+  ({ daysOfTheWeek = DAYS_OF_THE_WEEK, activatedDays = [] }, ref) => {
+    return (
+      <ToggleGroup
```

Comment:
Nice job *but* I don't think these are actually supposed to be interactive 👀

Reply:
They are not interactive. activated days is an array of the active ones, the rest will be deactivated and you can't modify the state by clicking on them.

Toggle and ToggleGroup were the most similar shadCN components to what we needed. It was easier to create it, but I considered that by using the Toggle we could (if we wanted) also have the toggle feature in the future, so I made a call to use Toggle rather than creating a more basic, limited, but similar component.

Reply:
I understand the reasoning. Something feels off about using "disabled" elements just for displaying purposes in this scenario, as I don't see how it could become interactive in the future. It feels like cutting corners, no?

Reply:
I actually thought I was doing the opposite, creating this from scratch was much easier, it's just a container with a string, but having a more powerful component that we could use in the future made more sense. I imagine the workable days need to be set at some point right? Why not using the same one, but make it dynamic in that case? This component can work both ways.

I'm happy to get rid of it if my reasoning was wrong.

Reply:
I understand. No biggie, let's go for this and we can always change our minds later. I would say though this will need a few stories to document it. Maybe in the `Forms` section?

Reply:
Good point, I'll add it